src/components/form/RadioButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef } from 'react';
+
+import { RadioButton as DesignSystemRadioButton } from '@digdir/design-system-react';
+import type { RadioButtonProps } from '@digdir/design-system-react';
+
+import classes from 'src/components/form/RadioButton.module.css';
+
+const Card = ({ children }: { children: React.ReactNode }) => <div className={classes.card}>{children}</div>;
+
+export interface IAppRadioButtonProps extends RadioButtonProps {
+  showAsCard?: boolean;
+}
+
+export const RadioButton = ({ showAsCard = false, ...rest }: IAppRadioButtonProps) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+  if (showAsCard) {
+    return (
+      <Card>
+        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
```

Comment:
This element is only clickable for visual effects. A screen reader would only want to click the inner `input` element of the `DesignSystemRadioButton`.

Reply:
Maybe we could add this explanation in a comment in the code to help future maintainers 😉